diff -ur a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	2019-01-30 13:50:51.302303958 -0700
+++ b/CMakeLists.txt	2019-01-30 14:01:16.283392823 -0700
@@ -24,7 +24,7 @@
 set (CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/tools/cmake/Modules")
 set (CMAKE_THREAD_PREFER_PTHREAD TRUE)
 
-include (CTest)
+#include (CTest)
 include (CheckLanguage)
 include (CheckLibraryExists)
 include (CheckSymbolExists)
@@ -33,13 +33,13 @@
 find_package (OpenSSL)
 find_package (Threads)
 find_package (PythonInterp REQUIRED)
-find_package (SWIG)
+#find_package (SWIG)
 find_package (CyrusSASL)
 
-enable_testing ()
+#enable_testing ()
 
 # Set up runtime checks (valgrind, sanitizers etc.)
-include(tests/RuntimeCheck.cmake)  
+# include(tests/RuntimeCheck.cmake)  
 
 ## Variables used across components
 
@@ -272,7 +272,7 @@
 
 set (SYSINSTALL_BINDINGS OFF CACHE BOOL "If SYSINSTALL_BINDINGS is OFF then proton bindings will be installed underneath ${BINDINGS_DIR} and each user will need to modify their interpreter configuration to load the appropriate binding. If SYSINSTALL_BINDINGS is ON, then each language interpreter will be queried for the appropriate directory and proton bindings will be installed and available system wide with no additional per user configuration.")
 
-set (BINDING_LANGS PYTHON RUBY)
+#set (BINDING_LANGS PYTHON RUBY)
 
 foreach (LANG ${BINDING_LANGS})
   set (SYSINSTALL_${LANG} OFF CACHE BOOL "Install ${LANG} bindings into interpreter specified location.")
@@ -327,7 +327,7 @@
 endif()
 
 # Prerequisites for Go
-find_program(GO_EXE go)
+#find_program(GO_EXE go)
 mark_as_advanced(GO_EXE)
 if (GO_EXE)
   set (DEFAULT_GO ON)
